plugins/source/aws/CONTRIBUTING.md

```diff
@@ -39,7 +39,7 @@ The process to follow for adding a new table is:
    
    You may use a type assertion on `meta` to obtain a reference to your interface functions, e.g.:
    ```go
-   svc := meta.(*client.Client).Services().MyService
+   svc := c.Services().MyService
```

Comment:
I think this was correct before, or otherwise the `c` variable will need to be declared first:

```
	c := meta.(*client.Client)
	svc := c.Services().MyService
```

I'd prefer to keep it the way it was though as it's more succinct